Fix build pipeline errors "process cannot access the file DownloadSonarPlugin.dll"

#### Description

Our build pipeline fails frequently due to file locking:
```
 CSC : error CS2012: Cannot open 'D:\a\1\s\build\DownloadSonarPlugin\obj\Debug\net46\DownloadSonarPlugin.dll' for writing -- 'The process cannot access the file 'D:\a\1\s\build\DownloadSonarPlugin\obj\Debug\net46\DownloadSonarPlugin.dll' because it is being used by another process.' [D:\a\1\s\build\DownloadSonarPlugin\DownloadSonarPlugin.csproj] [D:\a\1\s\src\Integration.Vsix\Integration.Vsix.csproj]
```

The failure is usually resolved by re-running the pipeline.

--- a/src/Integration.Vsix/Integration.Vsix.csproj
+++ b/src/Integration.Vsix/Integration.Vsix.csproj
@@ -368,7 +368,7 @@
          Also, we are using the "Exec" task to invoke MSBuild.exe rather the "MSBuild" task when building
          DownloadSonarPlugin.csproj to avoid getting file-locking errors when rebuilding.
     -->
-    <Exec Command="&quot;$(MSBuildBinPath)\msbuild.exe&quot; ..\..\build\DownloadSonarPlugin\DownloadSonarPlugin.csproj /p:PluginUrl=$(PluginUrl) /p:TargetDirectory=&quot;$(CFamilyTargetDirectory)&quot; /p:UseSharedCompilation=false /t:Restore;Build;EmbedCFamilyFiles" LogStandardErrorAsError="true" IgnoreExitCode="false" />
+    <Exec Command="&quot;$(MSBuildBinPath)\msbuild.exe&quot; ..\..\build\DownloadSonarPlugin\DownloadSonarPlugin.csproj /p:PluginUrl=$(PluginUrl) /p:TargetDirectory=&quot;$(CFamilyTargetDirectory)&quot; /p:UseSharedCompilation=false /t:Restore;Build;EmbedCFamilyFiles /nr:true" LogStandardErrorAsError="true" IgnoreExitCode="false" />
 
     <!-- Copy the license file to the same directory -->
     <Copy SourceFiles=".\CFamily\LICENSE.txt" DestinationFolder="$(CFamilyTargetDirectory)" SkipUnchangedFiles="true" />

--- a/src/Integration.Vsix/Integration.Vsix.csproj
+++ b/src/Integration.Vsix/Integration.Vsix.csproj
@@ -368,7 +368,7 @@
          Also, we are using the "Exec" task to invoke MSBuild.exe rather the "MSBuild" task when building
          DownloadSonarPlugin.csproj to avoid getting file-locking errors when rebuilding.
     -->
-    <Exec Command="&quot;$(MSBuildBinPath)\msbuild.exe&quot; ..\..\build\DownloadSonarPlugin\DownloadSonarPlugin.csproj /p:PluginUrl=$(PluginUrl) /p:TargetDirectory=&quot;$(CFamilyTargetDirectory)&quot; /p:UseSharedCompilation=false /t:Restore;Build;EmbedCFamilyFiles /nr:true" LogStandardErrorAsError="true" IgnoreExitCode="false" />
+    <Exec Command="&quot;$(MSBuildBinPath)\msbuild.exe&quot; ..\..\build\DownloadSonarPlugin\DownloadSonarPlugin.csproj /p:PluginUrl=$(PluginUrl) /p:TargetDirectory=&quot;$(CFamilyTargetDirectory)&quot; /p:UseSharedCompilation=false /t:Restore;Build;EmbedCFamilyFiles /nr:false" LogStandardErrorAsError="true" IgnoreExitCode="false" />
 
     <!-- Copy the license file to the same directory -->
     <Copy SourceFiles=".\CFamily\LICENSE.txt" DestinationFolder="$(CFamilyTargetDirectory)" SkipUnchangedFiles="true" />

